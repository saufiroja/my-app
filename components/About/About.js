/* eslint-disable react/no-unescaped-entities */
import React from "react";
import SectionTitle from "../SectionTitle/SectionTitle";

export default function About() {
  return (
    <section className="bg-[#5956E9] xl:mt-[-7%] lg:mt-[-8%] md:mt-[-11%] sm:mt-[-13%] pb-20" id="about">
      <div className="w-8/12 container">
        <div className="">
          <SectionTitle className="text-white text-center my-20">
            What's on the impostor website
          </SectionTitle>
        </div>
        <div className="py-10">
          <SectionTitle className="text-white">ABOUT US</SectionTitle>
          <p className="leading-relaxed text-white mt-6">
            Lorem Ipsum is simply dummy text of the printing and typesetting
            industry. Lorem Ipsum has been the industry's standard dummy text
            ever since the 1500s, when an unknown printer took a galley of type
            and scrambled it to make a type specimen book. It has survived not
            only five centuries, but also the leap into electronic typesetting,
            remaining essentially unchanged. It was popularised in the 1960s
            with the release of Letraset sheets containing Lorem Ipsum passages,
            and more recently with desktop publishing software like Aldus
            PageMaker including versions of Lorem Ipsum.
          </p>
        </div>
        <div className="py-10">
          <SectionTitle className="text-white">FREE ONLINE GAMES</SectionTitle>
          <p className="leading-relaxed text-white mt-6">
            Contrary to popular belief, Lorem Ipsum is not simply random text.
            It has roots in a piece of classical Latin literature from 45 BC,
            making it over 2000 years old. Richard McClintock, a Latin professor
            at Hampden-Sydney College in Virginia, looked up one of the more
            obscure Latin words, consectetur, from a Lorem Ipsum passage, and
            going through the cites of the word in classical literature,
            discovered the undoubtable source.
          </p>
        </div>
        <div className="py-10">
          <SectionTitle className="text-white">HOW TO PLAY</SectionTitle>
          <p className="leading-relaxed text-white mt-6">
            It is a long established fact that a reader will be distracted by
            the readable content of a page when looking at its layout. The point
            of using Lorem Ipsum is that it has a more-or-less normal
            distribution of letters, as opposed to using 'Content here, content
            here', making it look like readable English. Many desktop publishing
            packages and web page editors now use Lorem Ipsum as their default
            model text, and a search for 'lorem ipsum' will uncover many web
            sites still in their infancy.
          </p>
        </div>
      </div>
    </section>
  );
}
