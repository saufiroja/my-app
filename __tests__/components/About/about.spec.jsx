import React from 'react'
import { render, screen } from '@testing-library/react'
import About from '../../../components/About/About'
import renderer from "react-test-renderer"

describe('About', () => {
    it('renders a heading', () => {
        render(<About />)

    const heading = screen.getByRole('heading', {
        name: "What's on the impostor website",
        name: "ABOUT US",
        name: "FREE ONLINE GAMES",
        name: "HOW TO PLAY",
        })

        expect(heading).toBeInTheDocument()
    })

    it('matches the snapshot', () => {
    const tree = renderer.create(<About />).toJSON()

    expect(tree).toMatchSnapshot()
    })
})