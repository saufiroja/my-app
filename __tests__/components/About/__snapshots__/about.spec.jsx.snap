// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About matches the snapshot 1`] = `
<section
  className="bg-[#5956E9] xl:mt-[-7%] lg:mt-[-8%] md:mt-[-11%] sm:mt-[-13%] pb-20"
  id="about"
>
  <div
    className="w-8/12 container"
  >
    <div
      className=""
    >
      <h2
        className="xl:text-4xl lg:text-3xl md:text-3xl sm:text-2xl font-body  text-white text-center my-20"
      >
        What's on the impostor website
      </h2>
    </div>
    <div
      className="py-10"
    >
      <h2
        className="xl:text-4xl lg:text-3xl md:text-3xl sm:text-2xl font-body  text-white"
      >
        ABOUT US
      </h2>
      <p
        className="leading-relaxed text-white mt-6"
      >
        Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
      </p>
    </div>
    <div
      className="py-10"
    >
      <h2
        className="xl:text-4xl lg:text-3xl md:text-3xl sm:text-2xl font-body  text-white"
      >
        FREE ONLINE GAMES
      </h2>
      <p
        className="leading-relaxed text-white mt-6"
      >
        Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source.
      </p>
    </div>
    <div
      className="py-10"
    >
      <h2
        className="xl:text-4xl lg:text-3xl md:text-3xl sm:text-2xl font-body  text-white"
      >
        HOW TO PLAY
      </h2>
      <p
        className="leading-relaxed text-white mt-6"
      >
        It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy.
      </p>
    </div>
  </div>
</section>
`;
